build_info/ddrescue.control

```diff
@@ -0,0 +1,40 @@
+Package: ddrescue
+Version: @DEB_DDRESCUE_V@
+Architecture: @DEB_ARCH@
+Maintainer: @DEB_MAINTAINER@
+Section: Utilities
+Priority: optional
+Description: GNU data recovery tool
```

Comment:
Add the [homepage](https://www.gnu.org/software/ddrescue/ddrescue.html) of the package to the Homepage field here.